finality-aleph/src/sync/service.rs

```diff
@@ -757,6 +760,17 @@ where
                     self.handle_own_block(block);
                 },
 
+                maybe_favourite_block_sender = self.favourite_block_request.next() => {
+                    let favourite_block_sender = maybe_favourite_block_sender.ok_or(Error::FavouriteRequestChannelClosed)?;
+                    let favourite = self.handler.favourite_block();
+                    if let Err(_) = favourite_block_sender.send(favourite) {
+                        warn!(
+                            target: LOG_TARGET,
+                            "Error sending favourite block on request."
```

Comment:
This sounds confusingly as if it might be about sync requests over the network. Perhaps "Failed to respond with favourite header to user." or something like that. Also eliminates the "Error" in a `warn`.

Reply:
ok, done